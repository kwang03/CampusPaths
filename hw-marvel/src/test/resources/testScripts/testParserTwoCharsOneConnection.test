#Tests the parser when there are two characters with just one connection

LoadGraph myGraph twoCharsOnePath.csv

ListNodes myGraph

ListChildren myGraph Char1

ListChildren myGraph Char2