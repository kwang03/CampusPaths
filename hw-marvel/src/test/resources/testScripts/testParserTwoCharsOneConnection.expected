#Tests the parser when there are two characters with just one connection

loaded graph myGraph

myGraph contains: Char1 Char2

the children of Char1 in myGraph are: Char2(Book1)

the children of Char2 in myGraph are: Char1(Book1)